src/applications/mhv-secure-messaging/containers/Compose.jsx

```diff
@@ -122,7 +172,7 @@ const Compose = () => {
 
   return (
     <>
-      {!draftType && (
+      {(!draftType || isLoading || allTriageGroupsBlocked === undefined) && (
```

Comment:
if his is just used to leverage analytics, do we need to block user's flow with a loader spinner ? Or can this be ignored and just fetched in the background while user continues to compose a message? 
<img width="100" src="https://github.com/user-attachments/assets/00d5e35e-f8b4-4043-b956-d562cc8fc5ca"/>

Reply:
hmm, do we have to? Can we just fetch and process things in the background?

Reply:
Sure!  we can try it out and see what the response times are like for now since this is just for analytics. 

This might change in a future PR and I am okay with that.